### PR TITLE
User registers account

### PR DESCRIPTION
**Pull request by mikecm1141**
#### Pivotal URL or Waffle Card number:
#3

#### What does this PR do?
Allows user to register an account

#### Where should the reviewer start?
index.js
style.css

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -81,6 +82,24 @@ const authenticateUser = (username, password) => {
     .catch(error => console.error({error}));
 }
 
+// ***** Register User *****
+
+const registerUser = (username, password, passwordConfirmation) => {
+  let formData = new FormData();
+
+  formData.append('email', username);
```
Comment (MacInnes) [inline lib/index.js]: Kinda confusing mixing email and username here

File: lib/index.js [hunk 2]
```diff
@@ -332,6 +351,25 @@ $('#login-form').submit(function(event) {
...
+$('#register-form').submit(function(event) {
+  $('#register-error').html('');
+
+  event.preventDefault();
+
+  let username = $('#user-register').val();
+  let password = $('#pass-register').val();
+  let passwordConfirmation = $('#confirm-pass-register').val();
+
+  if ((password === '') || (passwordConfirmation === '') || (username === '')) {
```
Comment (MacInnes) [inline lib/index.js]: I think this could be handled by raw html by adding the require tag to the input

Comment (mikecm1141) [reply lib/index.js]: Didn't know that existed, thanks

File: lib/index.js (hunk 1 above)
Comment (mikecm1141) [reply lib/index.js]: I will edit this for better clarity thanks

**Pull request merged**